nile-backend/src/main/java/com/tp/Nile/services/ReviewService.java

```diff
@@ -5,9 +5,11 @@
 import com.tp.Nile.exceptions.NullReviewIdException;
 import com.tp.Nile.exceptions.NullUserIdException;
 import com.tp.Nile.models.Review;
+import org.springframework.stereotype.Service;
 
 import java.util.List;
 
+@Service
```

Comment:
Not necessary to have @Service on these interfaces. The implementations have the annotation.